README.md

```diff
@@ -1,58 +1,103 @@
+  window.watsonAssistantChatOptions = {
+    integrationID: "YOUR_INTEGRATION_ID",
+    region: "YOUR_REGION",
+    serviceInstanceID: "YOUR_SERVICE_INSTANCE_ID",
+    onLoad: function(instance) {
+      instance.render();
+    },
 
-<!-- You can and should do something like the below to make sure loading the script is non-blocking.
+    // **** The important part. This creates an instance of your integration.
+    serviceDeskFactory: (parameters) => new MyServiceDesk(parameters.callback),
+  };
+  setTimeout(function(){const t=document.createElement('script');t.src='https://web-chat.global.assistant.dev.watson.appdomain.cloud/versions/' + (window.watsonAssistantChatOptions.clientVersion || 'latest') + '/WatsonAssistantChatEntry.js';document.head.appendChild(t);});
+</script>
+```
+
+### Integration with web chat, using the build process in this repo
+
+This repo can be used as an isolated tool to build and test your integration with your web application. The basic steps to use it are:
+
+1. Use the build process in this repo to build a javascript bundle of your integration. This will produce a `servicedesk.bundle.js` file. Each integration should have a README describing in detail the steps necessary to build it.
+2. Upload the `servicedesk.bundle.js` file to a hosting location where the file can be downloaded and included in your application.
+3. Add a script tag to your application's HTML file that will load your integration. This will expose a global `WebChatServiceDeskFactory` function that can be provided to web chat.
+4. Add the configuration option to web chat to use `WebChatServiceDeskFactory`.
+
+```html
+<!-- The servicedesk.bundle.js is the script this repo will generate. You are responsible for finding a place to host it. -->
 <script>
   window.WebChatServiceDeskFactory = {};
   setTimeout(function(){const t=document.createElement('script');t.src='YOUR_HOST/servicedesk.bundle.js';document.head.appendChild(t);});
 </script>
--->
 
-<!-- Regular web chat embed script -->
 <script>
+  // Regular web chat embed script.
   window.watsonAssistantChatOptions = {
     integrationID: "YOUR_INTEGRATION_ID",
     region: "YOUR_REGION",
     serviceInstanceID: "YOUR_SERVICE_INSTANCE_ID",
     onLoad: function(instance) {
       instance.render();
     },
-    // The function that this project exports.
+    // The function that this project exports and is contained in servicedesk.bundle.js.
     serviceDeskFactory: window.WebChatServiceDeskFactory,
   };
-  setTimeout(function(){const t=document.createElement('script');t.src='https://web-chat.global.assistant.watson.appdomain.cloud/loadWatsonAssistantChat.js';document.head.appendChild(t);});
+  setTimeout(function(){const t=document.createElement('script');t.src='https://web-chat.global.assistant.dev.watson.appdomain.cloud/versions/' + (window.watsonAssistantChatOptions.clientVersion || 'latest') + '/WatsonAssistantChatEntry.js';document.head.appendChild(t);});
```

Comment:
This line of sample code points to a dev URI for loading WebChat

Reply:
Oooo, very good eye!